scripts/sync-checksums.sh

```
@@ -0,0 +1,126 @@
+print(f"  bumped:    {binary_name} {old_sum[:12]}... → {new_sum[:12]}...")
+PY
+}
+
+echo ">> Syncing Package.swift checksums from $ZIP_DIR"
+
+missing=0
+updated=0
+
+while IFS='|' read -r binary_name zip_prefix; do
+    # Match files like RACommons-v0.20.0.zip, RACommons-v0.20.0-beta.zip, etc.
+    zip_file=$(ls "$ZIP_DIR"/${zip_prefix}-v*.zip 2>/dev/null | head -1 || true)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)